.sync.yml

```diff
@@ -24,43 +25,19 @@ appveyor.yml:
     - set: docker/debian-8
     - set: docker/centos-7
     - set: docker/centos-8
+    - set: docker/opensuse-15
+  docker_defaults:
+    dist: focal
+
+  branches:
+    - master
+  remove_branches:
+    - main
 
 .pdkignore:
   paths:
     - /.editorconfig
 
-Gemfile:
```

Comment:
There is a reason to remove this block?

Reply:
Restored the block, but the build still fails. Looks like the PDK 2.x release made some changes that I am in not aware of. I removed this to see if it would work with default.

Reply:
On last PDK update, I had similar problems. When I bump `puppet-module-*-system-r` to 1.0. Stay on `~> 0.4` Inside gem.

Reply:
Tried with ~> 0.4, but that fails on transitive dependency.I think I should just revert the PDK for now and add the openSuse. That is what I need anyway

Reply:
👍 

I will investiage the PDK Update in a different PR. I saw, GH Actions support is finally there.

Reply:
@jkroepke , Thanks, that will be perfect!